src/libData/AccountData/AccountStore.cpp

```diff
@@ -67,7 +67,8 @@ AccountStore::AccountStore() {
   }
   // EVM required to run on Lookup nodes too for view calls
   if (ENABLE_EVM) {
-    if (not ipcScillaInit && !LOOKUP_NODE_MODE) {
+    // TODO lookup nodes may also need it
+    if (not ipcScillaInit /*&& !LOOKUP_NODE_MODE*/) {
```

Comment:
This change seems wrong, as we should not change anything related to Scilla.

Reply:
Changes in this area were introduced for the work Kaustubh was doing on the API interface.

We have some states that the logic here doesn't quit match, these are :

Running as a Zilliqa process and not the Lookup Node or LOOKUP_NODE_MODE setting

Running as the isolatedServer

Running the testroutines as part of the build causes an invocation of this method and has requirements too.

Reply:
Kaustub's change were to enable EVM on seeds/lookups.  But Scilla should not be affected.